### PR TITLE
Update to SF 8.2 CU1

### PR DESCRIPTION
**Pull request by nicolas-garcia**
Hello,

Update to SF 8.2 CU1

https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_82CU1.md

Thank you @darinegan !

### CODE REVIEW

**Comment by darinegan** [comment]
Hi @nicolas-garcia 

Apologies for my delay; I've just returned from leave over the holidays. Firstly, thank you for your contribution and taking the time to share. I've reviewed the PR and have 1 item for your attention.

File: automatic/service-fabric-sdk/service-fabric-sdk.nuspec [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <!-- Read this before publishing packages to chocolatey.org: https://github.com/chocolatey/chocolatey/wiki/CreatePackages -->
     <id>service-fabric-sdk</id>
     <title>Service Fabric SDK</title>
-    <version>5.2.1235</version>
+    <version>5.2.1363</version>
```
Comment (darinegan) [inline automatic/service-fabric-sdk/service-fabric-sdk.nuspec:8]: Can you confirm whether this change in SDK version to 1363 has any dependency on the related "service-fabric" package version 1363 or can remain at 1235?

```xml
<dependencies>
    <dependency id="service-fabric" version="8.2.1235" />
</dependencies>
```

**Review by darinegan — CHANGES_REQUESTED** [review]

File: automatic/service-fabric-sdk/service-fabric-sdk.nuspec (hunk 1 above)
Comment (nicolas-garcia) [reply automatic/service-fabric-sdk/service-fabric-sdk.nuspec:8]: Hi @darinegan

Sorry for the delay, I was on holiday too :)
I think I missed this dependency at first. Good catch, thank you!

**Review by darinegan — APPROVED** [review]

**Comment by darinegan** [comment]
Thanks @nicolas-garcia for your work on this. I've approved and merged this PR. I also pushed this version to Chocolatey.org which is pending review.

**Pull request merged**